scripts/status_consistency.py

```diff
@@ -0,0 +1,36 @@
+from app import geo
+from app.firebase import db
+
+from app.constants import locationsTable, venuesTable, GPS_LOCATIONS
+
+# The casing of tripadvisor is not consistent across our Firebase scripts
+TA_PROVIDER = "tripadvisor"
+TA_DETAILS = "tripAdvisor"
+
+dryRun = True
+
+def updateStatus(center, radius_km, provider, detailsProvider, version):
+    location_table = db().child(locationsTable).get().val()
+    placeIDs = geo.get_place_ids_in_radius(center, radius_km, location_table)
+    print("number found {}".format(len(placeIDs)))
+
+    statusTable = db().child(venuesTable, "status").get().val()
+    placeDetails = db().child(venuesTable, "details").get().val()
+
+    count = 0
+    placeList = []
+    for placeID in placeIDs:
+        providers = placeDetails[placeID]["providers"]
+        if detailsProvider in providers and statusTable[placeID][provider] == -1:
+            count += 1
+            st = db().child(venuesTable, "status", placeID)
+            if not dryRun:
+                st.update({provider: version})
+            placeList.append(placeID)
```

Comment:
super-nit: I'd keep the debug lines together (`placeList` and `count`) – otherwise, it obfuscates the real changes that are taking place.